docs/foundation/Icons/Usage.stories.tsx

```diff
@@ -40,11 +41,19 @@ IconSize.parameters = {
 };
 
 export const CustomColors = () => (
-  <Bookmark
-    color={["brand", "secondary"]}
-    iconSize="M"
-    aria-label="Click to bookmark"
-  />
+  <div style={{ display: "flex", gap: 8 }}>
+    <Bookmark
+      color={["brand", "secondary"]}
```

Comment:
For staying as it was, now it should be:
```suggestion
      color={["secondary", "brand"]}
```

Also, why did you add more icons to the sample? I'm ok with it (it showcases more), but then I would add them to the equivalent sample on Sprites stories.

Reply:
I moved them from the `inverted` sample and added an extra one, but honestly they don't add much, and maybe even _clutters_ the story (we're using 1 icon per sample). I'll remove